src/Credentials/ServiceAccountJwtAccessCredentials.php

```diff
@@ -207,6 +207,19 @@ public function getClientName(callable $httpHandler = null)
         return $this->auth->getIssuer();
     }
 
+    /**
+     * Get the private key from the keyfile.
+     *
+     * In this case, it returns the keyfile's private_key key, needed for JWT signing.
+     *
+     * @param callable $httpHandler Not used by this credentials type.
+     * @return string
+     */
+    public function getPrivateKey(callable $httpHandler = null)
```

Comment:
Why are you passing in the `$httpHandler` as the first argument?

Reply:
Saw it in `ProjectIdProviderInterface` and `ProjectIdProviderInterface`, I assumed it's useful, should I remove it?

Reply:
Yes - it's only necessary if the method needs to make an http request

Reply:
ok, done and pushed.